<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_11">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="imageDisplay">
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter_10">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QPushButton" name="chooseFile">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>选择图片</string>
         </property>
        </widget>
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>缩放 </string>
          </property>
         </widget>
         <widget class="QSlider" name="scaled">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QCheckBox" name="boundaryCheck">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>97</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>描边</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="skeletonCheck">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>97</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>写字</string>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>边界阈值1</string>
          </property>
         </widget>
         <widget class="QSlider" name="threshold1">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>20</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>边界阈值2</string>
          </property>
         </widget>
         <widget class="QSlider" name="threshold2">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>20</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="spawnLine">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>生成路径</string>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLineEdit" name="port">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>94</width>
           <height>31</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="netConnect">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>94</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>网络连接</string>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QComboBox" name="com">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>94</width>
           <height>31</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="comConnect">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>94</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>串口连接</string>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Code Output</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="codeOutput">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>169</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>114514</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="beginDraw">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>开始绘画</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
